### DISCUSSION TITLE
where can I find a compete lammps input file for the examples/water case?

### DISCUSSION THREAD

**Discussion by tonystarkiss** [deepmodeling/deepmd-kit#697] [Q&A]
I follow the step of guide, got graph.pb file,  and try to run MD with LAMMPS, the guide only specify the parameter:
**_pair_style     deepmd graph.pb
pair_coeff_**
where can I find a complete input file? actually I test my input file, it is not work.

**Comment by njzjz** [comment]
https://github.com/deepmodeling/deepmd-kit/blob/v1.3.3/examples/water/lmp/in.lammps
